themes/bootstrap3/templates/header.phtml

```diff
@@ -1,5 +1,5 @@
 <?php $account = $this->auth()->getManager(); ?>
-<div class="banner container navbar">
+<div class="banner container navbar <?= $this->stickyElement('navbar-banner') ?>" data-sticky-pos="1">
```

Comment:
It seems slightly less than ideal that we use a view helper to display a sticky class, but we always include the data element. A few possibilities:

1.) You could capture the sticky class to a variable, and then only output the data-sticky-pos if that value is non-empty.

2.) You could change stickyElement to just return a boolean, wrap it around the data-sticky-pos attribute, and associate sticky styles with the presence of the data element instead of with a specific class. (Though I think this may have negative performance implications).

3.) You could refactor the impacted elements to use the [HtmlAttributes view helper](https://docs.laminas.dev/laminas-view/v2/helpers/html-attributes/) instead of raw HTML for rendering attributes, and then you could change the stickyElement helper so that it injects multiple attributes into the HtmlAttributes result instead of directly into markup.

Reply:
I prefer that the ViewHelper returns the class. That way it is easier to change the class if necessary. But now the ViewHelper also returns the data attribute.

Reply:
If there are no functional implications in having the extra whitespaces I would prefer not to change this. I also think it would be nice but if that means that we have to make the code more complicated or harder to read it's not worth it in my opinion.

Reply:
There are no functional implications. I think the best way to solve this, if we wanted to solve it, would be to use the htmlAttributes view helper instead of assembling the code by hand... but whether that's better than the current approach is a matter of opinion -- depends on how much abstraction you prefer to see. The htmlAttributes approach will generate slightly tidier HTML output and is less prone to human error, but its extra layer of abstraction might negatively impact code readability. I don't insist on any changes, but that's probably the best one to consider, if changes are to be made at all.

Reply:
I wonder if a good way to split the difference is to encode the data attribute into a class and use [`extractClassParams`](https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/js/common.js#L432-L446) to get that data. For example, the helper would return `sticky sticky-pos:1`.

Reply:
I did some refactoring using the htmlAttributes view helper now